po/de.po

```diff
@@ -219,161 +253,166 @@ msgstr ""
 msgid "PDF files"
 msgstr "PDF-Dateien"
 
-#: pdfarranger/pdfarranger.py:468
+#: pdfarranger/pdfarranger.py:501
 msgid "All files"
 msgstr "Alle Dateien"
 
-#: pdfarranger/pdfarranger.py:473
+#: pdfarranger/pdfarranger.py:506
 msgid "Supported image files"
 msgstr "Unterstützte Bilddateien"
 
-#: pdfarranger/pdfarranger.py:693
+#: pdfarranger/pdfarranger.py:726
 msgid "Rendering…"
 msgstr "Vorschau erstellen …"
 
-#: pdfarranger/pdfarranger.py:771
+#: pdfarranger/pdfarranger.py:804
 msgid "untitled"
 msgstr "unbenannt"
 
-#: pdfarranger/pdfarranger.py:918 pdfarranger/pdfarranger.py:927
-#: pdfarranger/pdfarranger.py:1027 pdfarranger/pdfarranger.py:1052
-msgid "_Cancel"
-msgstr "_Abbrechen"
-
-#: pdfarranger/pdfarranger.py:926
+#: pdfarranger/pdfarranger.py:959
 msgid "Your changes will be lost if you don’t save them."
 msgstr "Änderungen gehen verloren, wenn sie nicht gespeichert werden."
 
-#: pdfarranger/pdfarranger.py:927
+#: pdfarranger/pdfarranger.py:960
 msgid "Do_n’t Save"
 msgstr "_Nicht speichern"
 
-#: pdfarranger/pdfarranger.py:927 pdfarranger/pdfarranger.py:1026
+#: pdfarranger/pdfarranger.py:960 pdfarranger/pdfarranger.py:1060
 #: data/menu.ui.h:4
 msgid "_Save"
 msgstr "_Speichern"
 
-#: pdfarranger/pdfarranger.py:936
+#: pdfarranger/pdfarranger.py:969
 msgid "Discard changes and close?"
 msgstr "Änderungen verwerfen und schließen?"
 
-#: pdfarranger/pdfarranger.py:937 data/menu.ui.h:45
+#: pdfarranger/pdfarranger.py:970 data/menu.ui.h:46
 msgid "_Close"
 msgstr "_Schließen"
 
-#: pdfarranger/pdfarranger.py:942
+#: pdfarranger/pdfarranger.py:975
 msgid "Save changes to “{}” before closing?"
 msgstr "Änderungen an „{}“ vor dem Schließen speichern?"
 
-#: pdfarranger/pdfarranger.py:945
+#: pdfarranger/pdfarranger.py:978
 msgid "Save changes before closing?"
 msgstr "Änderungen vor dem Schließen speichern?"
 
-#: pdfarranger/pdfarranger.py:970
+#: pdfarranger/pdfarranger.py:1004
 msgid "Discard changes and quit?"
 msgstr "Änderungen verwerfen und beenden?"
 
-#: pdfarranger/pdfarranger.py:971 data/menu.ui.h:46
+#: pdfarranger/pdfarranger.py:1005 data/menu.ui.h:47
 msgid "_Quit"
 msgstr "Been_den"
 
-#: pdfarranger/pdfarranger.py:976
+#: pdfarranger/pdfarranger.py:1010
 msgid "Save changes to “{}” before quitting?"
 msgstr "Änderungen an „{}“ vor dem Beenden speichern?"
 
-#: pdfarranger/pdfarranger.py:979
+#: pdfarranger/pdfarranger.py:1013
 msgid "Save changes before quitting?"
 msgstr "Änderungen vor dem Beenden speichern?"
 
-#: pdfarranger/pdfarranger.py:1021
+#: pdfarranger/pdfarranger.py:1055
 msgid "Save As…"
 msgstr "Speichern unter…"
 
-#: pdfarranger/pdfarranger.py:1021
+#: pdfarranger/pdfarranger.py:1055
 msgid "Export…"
 msgstr "Exportieren …"
 
-#: pdfarranger/pdfarranger.py:1051 data/menu.ui.h:1
+#: pdfarranger/pdfarranger.py:1098
+#, python-format
+msgid "A file named \"%s\" already exists. Do you want to replace it?"
+msgstr ""
+"Eine Datei mit dem Namen \"%s\" existiert bereits. Möchten Sie sie ersetzen?"
```

Comment:
I think "Sie diese ersetzen" is more common and smoother.

Reply:
Thanks for pointing this out.